Tribler/Test/Community/Multichain/StatisticsDispaly/test_statistics_database.py

```diff
@@ -0,0 +1,73 @@
+from Tribler.Test.test_as_server import BaseTestCase
+from Tribler.community.multichain.StatisticsDisplay.networkNode import NetworkNode
+from Tribler.community.multichain.StatisticsDisplay.dbDriver import DbDriver
+
+from twisted.internet.defer import inlineCallbacks
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+
+class TestStatisticsDatabase(BaseTestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestStatisticsDatabase, self).__init__(*args, **kwargs)
+
+    @blocking_call_on_reactor_thread
+    @inlineCallbacks
+    def setUp(self, **kwargs):
+        yield super(TestStatisticsDatabase, self).setUp()
+        self.driver = DbDriver()
+        self.focus_node = NetworkNode("30", self.driver)  # hardcoded hex of focus node
+
+
+    @blocking_call_on_reactor_thread
+    def test_get_neighbors(self):
+        # Arrange
```

Comment:
Arrange is the setup for the test. I based this test case on the test case found in:
Tribler/Test/Community/Multichain/test_multichain_database.py